exchanges/huobi/huobi_wrapper.go

```diff
@@ -176,7 +176,7 @@ func (h *HUOBI) SetDefaults() {
 		exchange.RestSpot:         huobiAPIURL,
```

Comment:
Could you please add as it takes a while for initial sync to finish?

```diff
diff --git a/exchanges/huobi/huobi_wrapper.go b/exchanges/huobi/huobi_wrapper.go
index a27a16e79..9f0f8500e 100644
--- a/exchanges/huobi/huobi_wrapper.go
+++ b/exchanges/huobi/huobi_wrapper.go
@@ -80,6 +80,16 @@ func (h *HUOBI) SetDefaults() {
                log.Errorln(log.ExchangeSys, err)
        }

+       err = h.DisableAssetWebsocketSupport(asset.Futures)
+       if err != nil {
+               log.Errorln(log.ExchangeSys, err)
+       }
+
+       err = h.DisableAssetWebsocketSupport(asset.CoinMarginedFutures)
+       if err != nil {
+               log.Errorln(log.ExchangeSys, err)
+       }
+
        h.Features = exchange.Features{
                Supports: exchange.FeaturesSupported{
                        REST:      true,
```

Reply:
Note: I corrected all of these somewhere recently in #1735 - That error would not have enough context :) 

Fixed https://github.com/gbjk/gocryptotrader/commit/fde0ad954